languages/elixir/exercises/concept/conditionals/lib/log_level.ex

```diff
@@ -0,0 +1,9 @@
+defmodule LogLevel do
+  def to_label(level) do
+    raise "to_label/1 not yet implemented"
```

Comment:
As we want this message to be as clear as possible to the student, maybe do:

```suggestion
    raise "Please implement the to_label/1 method"
```

If you like this, the same comment applies to the other stub method.